src/inbox/mod_inbox_rdbms_mssql.erl

```diff
@@ -49,10 +37,7 @@ set_inbox_incr_unread(Username, Server, ToBareJid, Content, MsgId, Timestamp) ->
 build_query(Username, Server, ToBareJid, Content, increment, MsgId, Timestamp) ->
```

Comment:
This function may be merged with `build_query/8` and actually changed to `build_query/6`.